### PR TITLE
Session and cookies

### PR DESCRIPTION
**Pull request by Baroka-wp**
In this PR wi implement User Authenficatiion feature 

- [ ] Remove current_user method in ApplicationController, devise will provide us with one.
- [ ] [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
- [ ] Can register a new user.
- [ ] User logs in with a combination of email and password.
- [ ] Hashed password should be stored in the database.
- [ ] Ask for confirmation of email.
- [ ] Can reset password.
- [ ] [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and  styling.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   after_initialize :init
```
Comment (OLIPLICHE) [inline app/models/user.rb:6]: - Hi Good Job for Displaying the login page. Kindly make sure the user can **log in** with a combination of email and password. 
- Login page
Capture
- server (data)
-

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Baroka-wp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linter result is green ✔️
- Commit messages are meaningful✔️
- The pull request have a descriptive title telling you what changes are inside ✔️
- The pull request have a short summary describing introduced changes in more detail ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
-  Please Update the Title of the Project. Remove **Hello Rails** and add **Blog app**. This  need to be done in your ReadMe file.
- **NB:** This comment above is not an inline comment because you did not display the ReadMe file in your PR

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Baroka-wp) [reply app/models/user.rb:6]: Hello @OLIPLICHE 
Please, when you add user, you should confirme the email. 
Please follow this link to open the mailler `http://localhost:3000/letter_opener`

Sorry, i should mention it at the submission

**Comment by Baroka-wp** [comment]
Please, when you add a user, you should confirm the email.
Please follow this link to open the Mailer http://localhost:3000/letter_opener

**Comment by OLIPLICHE** [comment]
Okay, thank you for your clarification. The Next Reviewer will read it.

**Review by danielufeli — APPROVED** [review]
Hi @Baroka-wp,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Baroka-wp** [comment]
## Thank you very much @danielufeli and @OLIPLICHE

**Pull request merged**